src/components/select-utils.js

```diff
@@ -63,23 +63,23 @@ export const joinPostData = state => postId => {
 
   // Get the list of post's recipients
   const recipients = post.postedTo
-    .map((subscriptionId) => {
-      let userId = (state.subscriptions[subscriptionId]||{}).user
-      let subscriptionType = (state.subscriptions[subscriptionId]||{}).name
-      if (userId === post.createdBy && subscriptionType === 'Directs') {
-        // Remove "directs to yourself" from the list
-        return false
-      } else {
-        return userId
-      }
+    .map(subscriptionId => {
+      const userId = (state.subscriptions[subscriptionId]||{}).user
+      const subscriptionType = (state.subscriptions[subscriptionId]||{}).name
+      return userId !== post.createdBy || subscriptionType !== 'Directs' ? userId : false
```

Comment:
I'm sorry, seems like I should clarify what I said above. There's a comment in the code, this one:
```
       // Remove "directs to yourself" from the list
```

Your change removes it. I think we should leave it there, because otherwise this piece of code:
```
userId !== post.createdBy || subscriptionType !== 'Directs'
```
will be completely weird for anyone reading the code a month from now. (And I know it's API's fault, not client's.)

So my suggestion is to **not** remove that comment.